# Study 909 — Preferred Reset Premium — the teardown

The excess-of-cash Sharpe race, the (variable − fixed) spread with its Newey-West *t*, the block-bootstrap CIs, the 2022 era cut, the equity-like drawdowns, the costed isolation spread, the rising-rate regime-switch, and the 20-seed synthetic control. Frozen real numbers mirror `docs/results.md` (fingerprint `15c5bee54e98`); the live cell runs the synthetic control only.

## 1. Frozen real-tape headline — flagship VRP vs PFF, excess-of-cash

In [1]:
R = {'start': '2014-06-30', 'end': '2026-06-30', 'n': 145, 'fingerprint': '15c5bee54e98', 'vrp_sharpe': 0.45, 'pff_sharpe': 0.23, 'adv': 0.23, 'spread_full': 1.28, 't_nw_full': 1.44, 't_1s_full': 1.01, 'boot_spread_lo': -0.48, 'boot_spread_hi': 3.07, 'boot_adv_lo': -0.0, 'boot_adv_hi': 0.57, 'era_low_spread': -0.27, 'era_low_t': -0.28, 'era_low_n': 91, 'era_high_spread': 3.9, 'era_high_t': 2.41, 'era_high_adv': 0.4, 'era_high_n': 54, 'var_ann': 4.88, 'var_dd': -16.7, 'var_sharpe': 0.4, 'fixed_ann': 4.46, 'fixed_dd': -59.4, 'fixed_sharpe': 0.19, 'cy2022_var': -11.4, 'cy2022_fix': -19.5, 'sleeve_var_sharpe': 0.39, 'sleeve_fix_sharpe': -0.04, 'sleeve_adv': 0.43, 'sleeve_spread': 3.21, 'sleeve_t': 2.49, 'sleeve_n': 73, 'cost8_net': 0.72, 'cost8_sharpe': 0.16, 'cost8_t': 0.81, 'cost4_net': 0.8, 'switch_sharpe': 0.26, 'always_var_sharpe': 0.4, 'always_fix_sharpe': 0.21, 'planted_t': 3.11, 'planted_high': 17.67, 'planted_low': -1.1, 'null_mean_t': -0.07, 'null_sd': 1.1, 'null_fire': 1}
print('window %s -> %s  (n=%d months)' % (R['start'], R['end'], R['n']))
print('VRP excess Sharpe %+.2f  vs  PFF excess Sharpe %+.2f  -> advantage %+.2f'
      % (R['vrp_sharpe'], R['pff_sharpe'], R['adv']))
print('(VRP-PFF) spread %+.2f%%/yr  NW t %+.2f  1s t %+.2f'
      % (R['spread_full'], R['t_nw_full'], R['t_1s_full']))
print('bootstrap spread 95%% CI [%+.2f, %+.2f]  (includes 0)'
      % (R['boot_spread_lo'], R['boot_spread_hi']))
print('bootstrap Sharpe-adv 95%% CI [%+.2f, %+.2f]  (touches 0)'
      % (R['boot_adv_lo'], R['boot_adv_hi']))

window 2014-06-30 -> 2026-06-30  (n=145 months)
VRP excess Sharpe +0.45  vs  PFF excess Sharpe +0.23  -> advantage +0.23
(VRP-PFF) spread +1.28%/yr  NW t +1.44  1s t +1.01
bootstrap spread 95% CI [-0.48, +3.07]  (includes 0)
bootstrap Sharpe-adv 95% CI [-0.00, +0.57]  (touches 0)


## 2. The era cut — the premium is entirely in the high-rate regime

Split at 2022-01. Low-rate: nothing. High-rate: the spread's HAC *t* clears 2. The effect is **not** era-robust — it is a bet on the rate regime, which is why the Signal stamp is **Mixed**, not Real.

In [2]:
print('%-9s %5s %14s %8s %10s' % ('era','n','spread/yr','NW t','adv'))
print('%-9s %5d %13.2f%% %+8.2f %+10.2f'
      % ('low-rate', R['era_low_n'], R['era_low_spread'], R['era_low_t'], -0.03))
print('%-9s %5d %13.2f%% %+8.2f %+10.2f'
      % ('high-rate', R['era_high_n'], R['era_high_spread'], R['era_high_t'], R['era_high_adv']))

era           n      spread/yr     NW t        adv
low-rate     91         -0.27%    -0.28      -0.03
high-rate    54          3.90%    +2.41      +0.40


## 3. Equity-like drawdowns and the multi-name sleeve

Preferreds are junior credit: the fixed sleeve fell ~59% in the GFC. The variable sleeve is younger (post-2014), worst −16.7%, and its carry survives that. Adding PFFV (2020+) the sleeve advantage clears the bar too — but that window is *all* high-rate regime, so it is the same 2022+ story, not independent evidence.

In [3]:
print('variable sleeve: ann %+.2f%%  maxDD %+.1f%%  excess Sharpe %+.2f'
      % (R['var_ann'], R['var_dd'], R['var_sharpe']))
print('fixed sleeve   : ann %+.2f%%  maxDD %+.1f%% (2008-09)  excess Sharpe %+.2f'
      % (R['fixed_ann'], R['fixed_dd'], R['fixed_sharpe']))
print('multi-name sleeve 2020-07+ (n=%d): var Sh %+.2f, fix Sh %+.2f, adv %+.2f,'
      ' spread %+.2f%%/yr (NW t %+.2f)'
      % (R['sleeve_n'], R['sleeve_var_sharpe'], R['sleeve_fix_sharpe'],
         R['sleeve_adv'], R['sleeve_spread'], R['sleeve_t']))

variable sleeve: ann +4.88%  maxDD -16.7%  excess Sharpe +0.40
fixed sleeve   : ann +4.46%  maxDD -59.4% (2008-09)  excess Sharpe +0.19
multi-name sleeve 2020-07+ (n=73): var Sh +0.39, fix Sh -0.04, adv +0.43, spread +3.21%/yr (NW t +2.49)


## 4. Tradability — thin net, and timing hurts

The market-neutral long-variable / short-fixed isolation nets only ~+0.7%/yr after costs (Sharpe ~0.16, *t* < 1). And the rising-rate regime-switch **underperforms** simply holding variable — the timing just churns cost.

In [4]:
print('costed isolation spread (8 bps one-way): net %+.2f%%/yr  Sharpe %+.2f  NW t %+.2f'
      % (R['cost8_net'], R['cost8_sharpe'], R['cost8_t']))
print('regime switch      excess Sharpe %+.2f' % R['switch_sharpe'])
print('always-variable    excess Sharpe %+.2f  <- beats the switch' % R['always_var_sharpe'])
print('always-fixed       excess Sharpe %+.2f' % R['always_fix_sharpe'])

costed isolation spread (8 bps one-way): net +0.72%/yr  Sharpe +0.16  NW t +0.81
regime switch      excess Sharpe +0.26
always-variable    excess Sharpe +0.40  <- beats the switch
always-fixed       excess Sharpe +0.21


## 5. Synthetic control — the machinery is unbiased (live, offline)

Plant a regime-contingent reset premium; confirm the detector recovers it and never fires on the null. Run live below; a faithful-engine check only — never cited in support of the real-tape stamp.

In [5]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
import numpy as np
from pref_reset import data, strategy as st
planted = st.synthetic_detect(data.synthetic_world(edge=0.0030, seed=909))
print('planted: spread NW t = %+.2f  (high %+.1f%%/yr, low %+.1f%%/yr)'
      % (planted['t_nw'], planted['spread_high_ann_pct'], planted['spread_low_ann_pct']))
ts = [st.synthetic_detect(data.synthetic_world(edge=0.0, dur_hit=0.0, seed=909+s))['t_nw']
      for s in range(20)]
ts = np.array(ts)
print('null (20 seeds): mean t %+.2f (sd %.2f), |t|>=2 in %d/20'
      % (ts.mean(), ts.std(ddof=1), int((np.abs(ts) >= 2).sum())))

planted: spread NW t = +3.11  (high +17.7%/yr, low -1.1%/yr)
null (20 seeds): mean t -0.07 (sd 1.10), |t|>=2 in 1/20


## 6. Verdict

**Signal — Mixed.** The reset premium is real but **regime-contingent**: full-sample +1.28%/yr (NW *t* = +1.44, bootstrap CI across zero), splitting into -0.27%/yr (*t* = -0.28) low-rate vs +3.90%/yr (*t* = +2.41) high-rate. It does not hold across sub-eras.

**Tradability — Fragile.** The isolation spread nets only +0.72%/yr (Sharpe +0.16, *t* < 1); timing the regime underperforms holding variable. Real-but-thin, one hiking cycle of evidence — the bankable form is a structural tilt to variable-rate preferreds, not a timed trade.